llvm/test/CodeGen/AMDGPU/i1-to-bf16.ll

```diff
@@ -0,0 +1,292 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=amdgcn -mcpu=gfx942 | FileCheck %s -check-prefix=GFX942
```

Comment:
Should also have tests on other targets, particularly the ones without legal bf16